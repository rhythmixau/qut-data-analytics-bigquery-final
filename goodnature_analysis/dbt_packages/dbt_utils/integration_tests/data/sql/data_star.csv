version https://git-lfs.github.com/spec/v1
oid sha256:780c7a78a37aaa8c2da53ed0d23a1ecdf2e19ab78959a25ba613f758698da27d
size 42
