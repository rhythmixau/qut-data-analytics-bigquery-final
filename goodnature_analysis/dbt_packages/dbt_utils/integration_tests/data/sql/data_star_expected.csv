version https://git-lfs.github.com/spec/v1
oid sha256:4eb3efd957aa19936080b34c2c6c4cbe9ddb6fc40262442578048f5bd4218176
size 28
